src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -26,7 +27,9 @@ attributes:
 
 .Ephemeral Storage
 
-With this kind of storage files are mounted to emptyDir. As the name says, it is initially empty. When a Pod is removed from a node for any reason, the data in the emptyDir is deleted forever.  This means that you lose all your changes if you stop your workspace (or workspace pod restarts for any reason).
+With this kind of storage files are mounted to `emptyDir`. As the name says, it is initially empty. When a Pod is removed from a node for any reason, the data in the `emptyDir` is deleted forever.  This means that you lose all your changes if you stop your workspace (or workspace pod restarts for any reason). Here more about `emptyDir`:link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[Volumes:emptyDir]
```

Comment:
I would remove - `Here more about `emptyDir`:link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[Volumes:emptyDir]`

and add the link to the emptyDir to the original sentence:

With this kind of storage files are mounted to link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[`emptyDir`]

Reply:
done